images/bare-metal-packet-flows.svg

```diff
@@ -0,0 +1 @@
671875q-0.453125 0.6875 -1.484375 1.015625q0.46875 0.09375 0.90625 0.3125q0.4375 0.21875 0.765625 0.578125q0.328125 0.34375 0.515625 0.8125q0.1875 0.46875 0.1875 1.0625zm-2.109375 -5.234375q0 -0.359375 -0.109375 -0.671875q-0.109375 -0.328125 -0.390625 -0.5625q-0.265625 -0.234375 -0.734375 -0.375q-0.453125 -0.140625 -1.171875 -0.140625l-1.671875 0l0 3.734375l1.625 0q0.546875 0 1.0 -0.109375q0.453125 -0.125 0.765625 -0.375q0.328125 -0.25 0.5 -0.609375q0.1875 -0.375 0.1875 -0.890625zm0.390625 5.296875q0 -0.46875 -0.1875 -0.828125q-0.1875 -0.359375 -0.5625 -0.609375q-0.359375 -0.25 -0.890625 -0.390625q-0.515625 -0.140625 -1.171875 -0.140625l-1.65625 0l0 4.109375l1.703125 0q1.40625 0 2.078125 -0.515625q0.6875 -0.53125 0.6875 -1.625zm11.788544 3.046875q-1.421875 0.59375 -3.0 0.59375q-2.515625 0 -3.875 -1.5q-1.34375 -1.515625 -1.34375 -4.46875q0 -1.421875 0.359375 -2.578125q0.375 -1.15625 1.0625 -1.953125q0.703125 -0.8125 1.6875 -1.25q0.984375 -0.4375 2.203125 -0.4375q0.828125 0 1.53125 0.140625q0.71875 0.140625 1.375 0.421875l0 1.59375q-0.640625 -0.34375 -1.34375 -0.53125q-0.6875 -0.1875 -1.5 -0.1875q-0.84375 0 -1.53125 0.3125q-0.671875 0.3125 -1.15625 0.90625q-0.46875 0.59375 -0.71875 1.46875q-0.25 0.859375 -0.25 1.953125q0 2.3125 0.9375 3.5q0.9375 1.171875 2.75 1.171875q0.765625 0 1.46875 -0.171875q0.703125 -0.1875 1.34375 -0.515625l0 1.53125z" fill-rule="nonzero"/><path fill="#000000" d="m242.45871 281.71002l-7.46875 0l0 -1.46875l3.0625 0l0 -8.703125l-2.859375 1.546875l-0.578125 -1.34375l3.78125 -1.984375l1.40625 0l0 10.484375l2.65625 0l0 1.46875zm10.257294 0l-7.46875 0l0 -1.46875l3.0625 0l0 -8.703125l-2.859375 1.546875l-0.578125 -1.34375l3.78125 -1.984375l1.40625 0l0 10.484375l2.65625 0l0 1.46875zm6.2416687 -2.875q0.296875 0 0.578125 0.125q0.28125 0.125 0.484375 0.328125q0.203125 0.203125 0.3125 0.484375q0.125 0.28125 0.125 0.59375q0 0.3125 -0.125 0.59375q-0.109375 0.265625 -0.3125 0.46875q-0.203125 0.203125 -0.484375 0.3125q-0.28125 0.125 -0.578125 0.125q-0.328125 0 -0.609375 -0.125q-0.265625 -0.109375 -0.46875 -0.3125q-0.203125 -0.203125 -0.328125 -0.46875q-0.109375 -0.28125 -0.109375 -0.59375q0 -0.3125 0.109375 -0.59375q0.125 -0.28125 0.328125 -0.484375q0.203125 -0.203125 0.46875 -0.328125q0.28125 -0.125 0.609375 -0.125zm14.647919 -3.078125q0 1.375 -0.28125 2.5q-0.265625 1.125 -0.828125 1.9375q-0.546875 0.796875 -1.375 1.234375q-0.8125 0.4375 -1.9375 0.4375q-0.953125 0 -1.734375 -0.359375q-0.78125 -0.359375 -1.34375 -1.09375q-0.546875 -0.75 -0.859375 -1.90625q-0.296875 -1.15625 -0.296875 -2.75q0 -1.359375 0.28125 -2.484375q0.28125 -1.140625 0.828125 -1.9375q0.546875 -0.8125 1.375 -1.25q0.84375 -0.4375 1.9375 -0.4375q0.953125 0 1.734375 0.359375q0.796875 0.359375 1.34375 1.109375q0.5625 0.75 0.859375 1.90625q0.296875 1.15625 0.296875 2.734375zm-1.59375 0.0625q0 -0.3125 -0.03125 -0.609375q-0.015625 -0.3125 -0.046875 -0.609375l-5.046875 3.75q0.140625 0.46875 0.34375 0.875q0.21875 0.390625 0.515625 0.6875q0.296875 0.28125 0.671875 0.453125q0.375 0.1
```

Comment:
Looks like the old diagram; I think @song-jiang is sending over the new version

Do SVGs render well in all current browsers?

Should you remove the old image? (Or is it still in use for the older versions of the site?)

Reply:
OK, I'll look for that. Thanks! Yes, the SVG renders well and it's well supported now. I kept the old PNG in the older version directories for expediency.